packages/calcite-components-angular/projects/component-library/package.json

```diff
@@ -0,0 +1,26 @@
+{
+  "name": "@esri/calcite-components-angular",
+  "version": "1.10.0",
+  "peerDependencies": {
+    "@angular/common": "^16.2.0",
+    "@angular/core": "^16.2.0",
+    "@esri/calcite-components": "1.10.0"
+  },
+  "devDependencies": {
+    "@esri/calcite-components": "1.10.0"
+  },
+  "dependencies": {
+    "tslib": "2.3.0"
```

Comment:
Is this needed by Angular or can we update the `tsconfig.json` target to avoid this dependency?

Reply:
It is added by the angular cli so I'm pretty sure it's required